tests/ontopy_tests/test_ontology.py

```diff
@@ -0,0 +1,26 @@
+"""Test Ontology class methods."""
+
+from ontopy import get_ontology
+from ontopy.testutils import ontodir
+
+animal = get_ontology(ontodir / "animal.ttl").load()
+
+
+# if True:
+def test_find():
+    """Test find() method."""
+    m1 = (animal.chasing, animal.prefLabel, "chasing")
+    for domain in "ontology", "imported", "world":
```

Comment:
This does not sufficiently test various domains. If you use the mammal.ttl ontology instead you can check that you get different responses for things that are not in the domain. mammal.ttl imports animal.